mappings/net/minecraft/client/render/entity/EntityRenderer.mapping

```diff
@@ -14,7 +14,6 @@ CLASS net/minecraft/unmapped/C_oddtxnnw net/minecraft/client/render/entity/Entit
 		ARG 2 tickDelta
 	METHOD m_gkfhbwdx getPositionOffset (Lnet/minecraft/unmapped/C_vptppxob;)Lnet/minecraft/unmapped/C_vgpupfxx;
 	METHOD m_hknkoqwr getTexture (Lnet/minecraft/unmapped/C_vptppxob;)Lnet/minecraft/unmapped/C_ncpywfca;
-	METHOD m_idqrsxfa (Lnet/minecraft/unmapped/C_vptppxob;)F
```

Comment:
This is something to look into for the javadocs:
![image](https://github.com/user-attachments/assets/05d32be1-e06a-4f78-b00e-a5050c43be05)